packages/compiler/src/core/parser.ts

```diff
@@ -982,6 +1075,32 @@ function createParser(code: string | SourceFile, options: ParseOptions = {}): Pa
     return undefined;
   }
 
+  function parseScalarMembers(): readonly ScalarConstructorNode[] {
+    if (token() === Token.Semicolon) {
```

Comment:
I feel like this should be using the parseList helper? It also seems like the behavior of this with respect to empty statements is different from model members (e.g. today `model Foo { ; }` is an error).

Reply:
im doing that no below? This just scan for the `scalar foo;` or `scalar bar extends foo;` cases

Reply:
That does the same here [playground](https://cadlplayground.z22.web.core.windows.net/prs/3022/?c=c2NhbGFyIG15IHsKICA7Cn0%3D&e=%40typespec%2Fopenapi3&options=%7B%7D)